apps/admin/external-dns-2/external-dns-private.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: external-dns-private-2
+  namespace: admin
+  annotations:
+    flux.weave.works/automated: "false"
+spec:
+  releaseName: external-dns-private
+  chart:
+    spec:
+      chart: external-dns
+      version: 6.1.4
+      sourceRef:
+        kind: HelmRepository
+        name: external-dns
+        namespace: admin
+  interval: 1m
+  values:
+    policy: sync
+    podLabels:
+      aadpodidbinding: external-dns
+    provider: azure-private-dns
+    sources:
+      - ingress
+    domainFilters:
+      - service.core-compute-preview.internal
```

Comment:
Is this not preview specific and needs to be moved to patch ?

Reply:
I'll do that now